### PR TITLE
qhull: migrate from homebrew/science

### PR DESCRIPTION
**Pull request by fxcoudert**
See https://github.com/Homebrew/homebrew-core/issues/15619

Amended the test slightly to check its output

### CODE REVIEW

File: Formula/qhull.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    result = shell_output("#{bin}/rbox c D2 | #{bin}/qconvex s n 2>&1")
```
Comment (ilovezfs) [inline Formula/qhull.rb]: pipe_output would be preferable here

Comment (fxcoudert) [inline Formula/qhull.rb]: Why `pipe_output` over `shell_output`?

Comment (ilovezfs) [inline Formula/qhull.rb]: We could probably convert everything to just use shell_output, but `pipe_output` is intended for this case.
```
pipe_output("#{bin}/qconvex s n 2>&1", "#{bin}/rbox c D2", 0)
```

File: Formula/qhull.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Qhull < Formula
+  desc "Code for computing convex hulls in n dimensions"
```
Comment (ilovezfs) [inline Formula/qhull.rb]: maybe just "Computes" instead of "Code for computing"

**Pull request closed without merging**